app/jobs/application_job.rb

```diff
@@ -1,9 +1,37 @@
+require 'gds_api/exceptions'
+require 'google/apis/analyticsreporting_v4'
+
 class ApplicationJob
   include Sidekiq::Worker
 
+  # List of errors to retry without alerting Sentry until their final retry
+  ERRORS_TO_RETRY_WITHOUT_ALERTS = [
+    Google::Apis::RateLimitError,
```

Comment:
I would not include here `Google::Apis::RateLimitError`. It could have implications in our GA quota.